store/mockstore/mocktikv/cluster.go

```diff
@@ -340,6 +341,12 @@ func (c *Cluster) SplitIndex(mvccStore MVCCStore, tableID, indexID int64, count
 	c.splitRange(mvccStore, NewMvccKey(indexStart), NewMvccKey(indexEnd), count)
 }
 
+// SplitKeys evenly splits the start, end key into count regions.
```

Comment:
s/count regions/multiple regions?

Reply:
em... `count` means the parameter `count`. And https://github.com/pingcap/tidb/pull/10152/files/363de0e82d296f19cd2a8802f4dca4353ed70e4e#diff-02d62c73e299ef92b48ee63dfc939369R336 use the same comment, so I just be consistent with it.